_caseStudies/2020-05-10-united-nations-lgbtqi-staff-platform.md

```diff
@@ -104,28 +104,26 @@ One example was how to classify the word “consultant” in the UN system. Cons
 
 ### Community Building
 
-One aspect of the platform that was missing in the original design is a community building effort. [There are many open source tools](https://opensource.com/alternatives/slack) available that can be integrated into a platform to act as a free direct message or community comment discussion boards. It’s important to note that given the sensitive nature of the platform and the ever-present existence of online bullying and harassment, any direct message or community commenting feature would need strict governance rules and an active community to moderate. This would be a feature to build in a later iteration.
+One aspect of the platform that was missing in the original design is a community building effort. [There are many open source tools][os-alternatives-slack] available that can be integrated into a platform to act as a free direct message or community comment discussion boards. It’s important to note that given the sensitive nature of the platform and the ever-present existence of online bullying and harassment, any direct message or community commenting feature would need strict governance rules and an active community to moderate. This would be a feature to build in a later iteration.
 
 ## Conclusion
 
 While the intent of the platform was important and the problem statement clearly laid out, there was a missed opportunity to take advantage of open source tools in the design to navigate around the major MERL challenges and funding constraints. Open source software could have enabled the platform to use the rich information already collected from project interviews and increased data points without significant additional effort. Though the original momentum of the project has lapsed, there is opportunity to carry the important initiative forward.
 
 ## To Get Involved
 
-For more information or if you are interested in reprising the work, [please email the MERL Center](mailto:themerlcenter@gmail.com).
+For more information or if you are interested in reprising the work, [please email the MERL Center][email-merl-center].
+
+[email-merl-center]: mailto:themerlcenter@gmail.com
+[digital-security]: https://gisf.ngo/wp-content/uploads/2020/02/2224-EISF-2017-Digital-Security-of-LGBTQI-Aid-Workers-Awareness-and-Response.pdf
+[crowdsourcing-platform]: https://www.devex.com/news/can-an-interactive-online-map-better-inform-lgbtqi-aid-worker-security-88008
+[pybossa]: https://pybossa.com/
+[sensemaker]: https://thecynefin.co/about-sensemaker/
```

Comment:
Original url: https://sensemaker.cognitive-edge.com/product-category/sensemakerapplications/.

When accessing this link, it will redirect to https://thecynefin.co with a `404`. After searching through the page, the one in use: https://thecynefin.co/about-sensemaker/, looks relevant as the replacement.

Reply:
Yes, this is referenced and being addressed in PR #94